xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -94,11 +98,11 @@ CDVDSubtitlesLibass::CDVDSubtitlesLibass()
   ass_set_use_margins(m_renderer, 0);
   ass_set_font_scale(m_renderer, 1);
 
-  int fontconfig = settings->GetBool(CSettings::SETTING_SUBTITLES_OVERRIDEASSFONTS) ? 0 : 1;
   // libass uses fontconfig (system lib) which is not wrapped
   //  so translate the path before calling into libass
-  ass_set_fonts(m_renderer, CSpecialProtocol::TranslatePath(strPath).c_str(), "Arial", fontconfig,
-                nullptr, 1);
+  strPath = URIUtils::AddFileToFolder(strPath, forcedFont);
```

Comment:
Same again the line above this (although it's not in scope as it's not changed in this PR